packages/rn-tester/RNTesterPods.xcodeproj/project.pbxproj

```diff
@@ -589,7 +589,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export NODE_BINARY=node\nexport SOURCEMAP_FILE=../sourcemap.ios.map\n# export FORCE_BUNDLING=true\n\"$SRCROOT/../../scripts/react-native-xcode.sh\" $SRCROOT/../../packages/rn-tester/js/RNTesterApp.ios.js\n";
+			shellScript = "export NODE_BINARY=node\nexport SOURCEMAP_FILE=../sourcemap.ios.map\nexport FORCE_BUNDLING=true\nexport USE_HERMES=true\nexport PROJECT_ROOT=\"$SRCROOT/../../\"\nexport HERMES_PATH=\"$SRCROOT/../../node_modules/hermes-engine-darwin\"\n\"$SRCROOT/../../scripts/react-native-xcode.sh\" $SRCROOT/../../packages/rn-tester/js/RNTesterApp.ios.js\n";
```

Comment:
We are explicitly setting `PROJECT_ROOT` to `../../../` which is the root of the entire repository. This is how RNTester is bundled in `DEBUG` (running `yarn start` from the root) and w/o this setting, Metro will not work (at a RNTester level, inside `packages/rn-tester`, there is no `react-native` dependency declared and it can't find `ios` and `android` platforms).

Reply:
Before this quick fix, there was no `main.jsbundle` outputted from the `react-native-xcode.sh` w/o any proper error message.